docker-compose.test.yml

```diff
@@ -1,11 +1,50 @@
 version: '3'
 services:
+  audit:
+    image: node:10
```

Comment:
maybe we should use the actual built container...

Reply:
Yes, why the one from `docker-compose.yml` doesn't cut it?

Reply:
That is also a `node` image rather than the application one. Without it the tests don't cover the image that actually gets pushed? Things that can go wrong include file permissions and ownership, missing or additional files, wrong `CMD`, ...

Reply:
think it was because i wanted to add artificial awaits locally

Reply:
if we use liberoadmin/audit:latest, then wouldn't it use the previous built image. at this stage, the new image hasn't been built yet.

Reply:
Should use `libero/audit:${IMAGE_TAG}`; I would expect a `docker-compose build` to always happen before integration tests are run (even if unit tests are run locally)?

Reply:
The problem is that we then need to build on PR and not on master. Not sure of the implication of that. will revert for now and can revisit later